db/src/trie/sectriedb.rs

```diff
@@ -35,9 +35,9 @@ impl<'db> SecTrieDB<'db> {
     /// Initialise to the state entailed by the genesis block.
     /// This guarantees the trie is built correctly.
     /// Returns an error if root does not exist.
-    pub fn new(db: &'db HashDB, root: &'db H256) -> super::Result<Self> {
+    pub fn new_db(db: &'db HashDB, root: &'db H256) -> super::Result<Self> {
```

Comment:
Just use other word , like `create`, When we use `SecTrieDB::create`, we alread know that it will create/new a secTrieDB

Reply:
OK